projects/Canary/ClientApp/src/components/tests/EDRSRoundtripProducing.js

```diff
@@ -178,9 +178,15 @@ export class EDRSRoundtripProducing extends Component {
                   </Header>
                   <div className="p-b-10" />
                   <Getter updateRecord={this.updateRecord} allowIje={false} recordType={this.props.params.recordType} />
-                  <Issues issues={this.state.issues} />
+                  {/* <Issues issues={this.state.issues} /> */}
```

Comment:
PR looks good to me. Just a nitpick, am I right in assuming there's no need to keep this commented line for posterity?